hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -268,7 +268,7 @@
               - Ensure IP operation get back normal after on-the-fly reset finished
             '''
       stage: V2
-      tests: [""]
+      tests: ["i2c_target_ack_stop"]
```

Comment:
should the test be renamed as well to match?

Reply:
change the name to i2c_target_unexp_stop